ci-operator/step-registry/windows/e2e/operator/test/mirror-images/windows-e2e-operator-test-mirror-images-commands.sh

```diff
@@ -63,7 +63,9 @@ registry.access.redhat.com/ubi9/ubi-minimal:latest=MIRROR_REGISTRY_PLACEHOLDER/u
 registry.access.redhat.com/ubi8/httpd-24:1-299=MIRROR_REGISTRY_PLACEHOLDER/ubi8/httpd-24:1-299
 registry.k8s.io/sig-storage/csi-provisioner:v3.3.0=MIRROR_REGISTRY_PLACEHOLDER/sig-storage/csi-provisioner:v3.3.0
 registry.k8s.io/sig-storage/livenessprobe:v2.8.0=MIRROR_REGISTRY_PLACEHOLDER/sig-storage/livenessprobe:v2.8.0
+registry.k8s.io/sig-storage/livenessprobe:v2.9.0=MIRROR_REGISTRY_PLACEHOLDER/sig-storage/livenessprobe:v2.9.0
 k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.7.0=MIRROR_REGISTRY_PLACEHOLDER/sig-storage/csi-node-driver-registrar:v2.7.0
```

Comment:
will be removed in follow-up PR after https://github.com/openshift/windows-machine-config-operator/pull/2286 goes in